app/templates/components/modals/add-user-role-modal.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="header">
+  {{t 'Add Role'}}
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action (optional submit) on='submit' preventDefault=true}}>
+    <div class="field">
+      <label class="required">
+        {{t 'User Email'}}
+      </label>
+      {{input type='text' name='user_email'}}
+    </div>
+    <div class="field">
+      <label class="required">
+        {{t 'User role'}}
+      </label>
+      {{#ui-dropdown class='fluid selection'}}
+        {{input type='hidden' name='user_role'}}
+        <i class="dropdown icon"></i>
+        <div class="default text">
+          {{t 'Select a role'}}
+        </div>
+        <div class="menu">
+          <div class="item" data-value="track_organizer">
+            {{t 'Track Organizer'}}
+          </div>
+          <div class="item" data-value="coorganizer">
+            {{t 'Co-organizer'}}
+          </div>
+          <div class="item" data-value="moderator">
+            {{t 'Moderator'}}
+          </div>
+          <div class="item" data-value="registrar">
+            {{t 'Registrar'}}
+          </div>
+        </div>
+      {{/ui-dropdown}}
+    </div>
+  </form>
+</div>
+<div class="actions">
+  <button type="button" class="ui black button" {{action 'close'}}>
+    {{t 'Cancel'}}
+  </button>
+  <button type="button" class="ui green right labeled icon button" {{action 'addRole'}}>
```

Comment:
Instead of adding an action for this, can't you wrap all of this inside `form` element and add `type=submit` to the button and let the default form event handle it for us ?